target-chains/cosmwasm/tools/src/deployer/injective.ts

```diff
@@ -169,6 +170,38 @@ export class InjectiveDeployer implements Deployer {
     }
   }
 
+  async updateAdmin(newAdmin: string, contract: string): Promise<void> {
+    const currAdmin = this.injectiveAddress();
+
+    const updateAdminMsg = new MsgUpdateAdmin({
+      sender: currAdmin,
+      newAdmin,
+      contract,
+    });
+
+    await this.signAndBroadcastMsg(updateAdminMsg);
+    console.log(await this.getContractInfo(contract));
```

Comment:
nitpick: it would be better to do this logging in the code that calls `updateAdmin` instead of here. In the future, someone else may call this function, and they may not want this log message.

Reply:
(i realize you're working off the code that i wrote, which is also probably doing the logging in the wrong place, so that's my fault.)